server/helpers/helper.js

```diff
@@ -1,13 +1,38 @@
 import bcrypt from 'bcryptjs';
+import moment from 'moment';
+
+/**
+ * @description formats a date using moment.js
+ * @function
+ * @param {string} date The date to format
+ * @returns {Date} Date
+ */
+export function formatDate(date) {
```

Comment:
I'm follwing the Airbnb standard, which say prefer normal function for things like that